map/routing_mark.cpp

```diff
@@ -192,7 +192,8 @@ drape_ptr<df::UserPointMark::SymbolNameZoomInfo> RouteMarkPoint::GetSymbolNames(
       case 0: name = "route-point-a"; break;
       case 1: name = "route-point-b"; break;
       case 2: name = "route-point-c"; break;
-      default: name = ""; break;
+      default: name = "route-point-c"; break; // This should be replaced with the "+" icon
```

Comment:
Almost :) Sorry for bikeshedding, but we write // style comments on the line(s) before the code, not after it:

```
    // TODO: Properly add icons for other letters/numbers after C.
    default: name = "route-point-c"; break;
```

The comment about "+" is not needed, because 1) TODO mentions it and 2) the final solution may be different than using a "+".

Reply:
I understand, clean code is good code :).

I'm new to collaborating and I really appreciate your patience. I rewrote my comments.